src/App.css

```diff
@@ -181,7 +246,7 @@ li {
 }
 
 .book-title {
-  width: 42%;
+  width: 55%;
 }
 
 .current-chapter {
```

Comment:
Great design @flemton 🤝

- [ ] As stated in the project requirements, kindly ensure your design is a close replica of the design provided. Kindly ensure the `CURRENT CHAPTER` paragraph is well aligned just like in the design given and it is not overlapping other elements.
- Your design
![overlapping](https://user-images.githubusercontent.com/63917725/224523094-a0b91be5-ed0b-4d7d-b895-6ef7a292482c.PNG)
- Given design
![Given design](https://user-images.githubusercontent.com/63917725/224523107-a842ee3a-1162-4f51-9da7-87f2978e24f8.PNG)